backend-services/src/web_asgi/settings.py

```diff
@@ -12,3 +12,8 @@ class AppSettings(Settings):
     staging_mode: bool = False
     wallet_db_name: str
     wallet_db_connection_string: MongoDsn
+    autofund_account: str
+    autofund_key: str
+    autofund_server: str
```

Comment:
For real this time.  Well done catching my error!
```suggestion
    autofund_server: HttpUrl
```